agent/txn_endpoint_test.go

```diff
@@ -51,24 +53,40 @@ func TestTxnEndpoint_Bad_Size_Item(t *testing.T) {
          }
      }
  ]
- `, strings.Repeat("bad", 2*maxKVSize))))
-	req, _ := http.NewRequest("PUT", "/v1/txn", buf)
-	resp := httptest.NewRecorder()
-	if _, err := a.srv.Txn(resp, req); err != nil {
-		t.Fatalf("err: %v", err)
-	}
-	if resp.Code != 413 {
-		t.Fatalf("expected 413, got %d", resp.Code)
+ `, value)))
+		req, _ := http.NewRequest("PUT", "/v1/txn", buf)
+		resp := httptest.NewRecorder()
+		if _, err := agent.srv.Txn(resp, req); err != nil {
+			t.Fatalf("err: %v", err)
+		}
+		if resp.Code != 413 && !pass {
+			t.Fatalf("expected 413, got %d", resp.Code)
+		}
+		if resp.Code != 200 && pass {
+			t.Fatalf("expected 200, got %d", resp.Code)
+		}
 	}
+
+	t.Run("toobig", func(t *testing.T) {
+		a := NewTestAgent(t, t.Name(), "")
+		testIt(a, false)
+		a.Shutdown()
+	})
+
+	t.Run("allowed", func(t *testing.T) {
+		a := NewTestAgent(t, t.Name(), "")
+		testIt(a, false)
```

Comment:
Shouldn't `pass` be `true` in this case? It seems to be identical to the `toobig` test above currently.

Reply:
...yes. Will fix.